docs/tools/vdb_table/vendor.schema.json

```diff
@@ -40,8 +40,8 @@
     "vector_dims": {"allOf": [{"$ref": "#/$defs/integerWithSource"}], "$comment": "Ops | Vector Dims | Maximum number of vector dimensions." },
     "github_stars": {"allOf": [{"$ref": "#/$defs/integerWithSource"}], "$comment": "Stats | GitHub ⭐ | The number of stars for the core product repository." },
```

Comment:
Make these also statsWithSource - so that if there's a way to find last 90days, or update stats representation, it works for all.

Reply:
Also lines 13-16 have duplicates for these properties. Let's remove lines 13-16

Reply:
okay